### ISSUE TITLE
SSL disabled for Redshift connections

### ISSUE THREAD

**Issue by johnchenghk01** [metabase/metabase#13261]
**Describe the bug**
The SSL option is disabled in v0.36.5.1. However, it works fine in v0.35.4
Our Redshift enforces users to connect with SSL, thus all connections to redshift are broken.

**Logs**
```
FATAL: no pg_hba.conf entry for host "::ffff:x.x.x.x", user "metabase", database "data_warehouse", SSL off
```

**Information about your Metabase Installation:**
- Metabase version: v0.36.5.1

**Severity**
T2
We downgraded metabase for now.

**Comment by jeff303** [comment]
@johnchenghk01 , were you able to verify whether the fixes under #13304 fixed this one, or if it's still occurring?

**Comment by flamber** [comment]
Closing as this has never been reported by anyone else and was most likely fixed in 0.36.7
If anyone is still seeing this problem, then include "Diagnostic Info" from Admin > Troubleshooting, and the logs from Admin > Troubleshooting > Logs, and steps-to-reproduce.
